### PR TITLE
Tweaks, updates and fixes

### PR DESCRIPTION
**Pull request by Jolly-66**
Contributing file is updated, so people can stop filling it out (why would you even do it knowing we don't have a CL, yet at least)

Mob move speed has been decreased by 0.5. 
Also for once, Goof wasn't smoking crack, increased min age to 18 and max to 300 (may lower if its a problem).
Stat panel clock is fixed; No more 23:00:00. Comment there shouldn't be touched, or I WILL kill you.
Extended should be brought back. Now people can shut the fuck up about black orbit when Dynamic can't spend anything on anything.
Night shifts are on (didn't know this was a config)

### CODE REVIEW

**Comment by Jolly-66** [comment]
also sec borgs are nuked fuck you

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -303,8 +303,8 @@
 #define OFFSET_NECK "neck"
 
 //MINOR TWEAKS/MISC
-#define AGE_MIN 17 //youngest a character can be
-#define AGE_MAX 85 //oldest a character can be
+#define AGE_MIN 18 //youngest a character can be, NON-MODULE START
+#define AGE_MAX 300 //oldest a character can be, NON-MODULE END
```
Comment (MrMelbert) [inline code/__DEFINES/mobs.dm:307]: what is he planning?

File: config/lavaruinblacklist.txt [hunk 2]
```diff
@@ -26,7 +26,7 @@ _maps/RandomRuins/LavaRuins/lavaland_surface_sloth.dmm
 #_maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm
 #_maps/RandomRuins/LavaRuins/lavaland_surface_ww_vault.dmm
 #_maps/RandomRuins/LavaRuins/lavaland_surface_automated_trade_outpost.dmm
-#_maps/RandomRuins/LavaRuins/lavaland_surface_xeno_nest.dmm
+_maps/RandomRuins/LavaRuins/lavaland_surface_xeno_nest.dmm
```
Comment (MrMelbert) [inline config/lavaruinblacklist.txt:29]: this is kinda sad

File: config/spaceruinblacklist.txt [hunk 3]
```diff
@@ -3,7 +3,7 @@
 #A list of maps valid to blacklist can be found in _maps\RandomRuins\SpaceRuins
 #SPECIFYING AN INVALID MAP WILL RESULT IN RUNTIMES ON GAME START
 
-#_maps/RandomRuins/SpaceRuins/abandonedteleporter.dmm
+_maps/RandomRuins/SpaceRuins/abandonedteleporter.dmm
```
Comment (MrMelbert) [inline config/spaceruinblacklist.txt:6]: this is very sad

File: config/spaceruinblacklist.txt [hunk 4]
```diff
@@ -35,18 +35,18 @@ _maps/RandomRuins/SpaceRuins/hilbertshoteltestingsite.dmm
...
+_maps/RandomRuins/SpaceRuins/spacehotel.dmm
 _maps/RandomRuins/SpaceRuins/thederelict.dmm
-#_maps/RandomRuins/SpaceRuins/turretedoutpost.dmm
+_maps/RandomRuins/SpaceRuins/turretedoutpost.dmm
 #_maps/RandomRuins/SpaceRuins/vaporwave.dmm
 #_maps/RandomRuins/SpaceRuins/way_home.dmm
-#_maps/RandomRuins/SpaceRuins/whiteshipdock.dmm
-#_maps/RandomRuins/SpaceRuins/whiteshipruin_box.dmm
+_maps/RandomRuins/SpaceRuins/whiteshipdock.dmm
+_maps/RandomRuins/SpaceRuins/whiteshipruin_box.dmm
```
Comment (MrMelbert) [inline config/spaceruinblacklist.txt:49]: ...whiteship disabled?

**Pull request merged**